src/styles/blocks/month.scss

```
@@ -0,0 +1,65 @@
+.month {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  gap: $gaps;
+
+  width: monthSize($day-size);
+
+  padding: 30px;
+
+  &__day {
+    box-sizing: border-box;
+    position: relative;
+    @include square($day-size);
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    transition-duration: 0.5s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

also, create variable for transition duration
```suggestion
    transition: transform $transitionDuration;
```